maventa/maventa.ts

```diff
@@ -0,0 +1,95 @@
+import https from 'https';
+import { getAccessToken } from './auth';
+import config from './config';
+import Invoice from './types/invoice';
+
+// Send company invoice dependencies
+
+// import fs from 'fs';
+// import FormData from 'form-data';
+
+export async function listInvoices(): Promise<Invoice[]> {
+  return new Promise((resolve, reject) => {
+    try {
+      getAccessToken().then(async (token) => {
+        const options = {
```

Comment:
Same here, no need to use NodeJS only dependency. Use our HTTP libraries. These are easier to unit test as well.